test/setContent.js

```diff
@@ -0,0 +1,21 @@
+import test from 'ava';
+import Config from '../';
+
```

Comment:
Create an extra test that fails when no string (`undefined`, a number, ...) is provided. [Here](https://github.com/SamVerschueren/cordova-config/blob/master/test/parse.js#L8-L10) is an example.